Changelog.md

```diff
@@ -13,6 +13,7 @@ Bugfixes:
  * ABIEncoderV2: Fix internal error related to ecrecover.
  * ABIEncoderV2: Fix internal error related to mappings as library parameters.
  * Inline Assembly: Proper error message for missing variables.
+ * Optimizer: Fix bug related to unused tag removal across assemblies.
```

Comment:
If this never had any adverse effect we should mention that.

Reply:
Reworded into mentioning "internal error", I think this should be sufficient. It always resulted in a "tag not found" - tag numbers are not re-used, only replaced.

Reply:
I think we should still mention it didn't generate invalid code to avoid scaring people.

Reply:
Added.